examples/web-chat-ai-sdk/src/style.css

```
@@ -0,0 +1,142 @@
+  display: grid;
+  grid-template-columns: minmax(0, 1fr) auto;
+  gap: 10px;
+  max-width: 720px;
+}
+
+.composer label {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.11.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)